bulbs/content/tasks.py

```diff
@@ -199,7 +201,9 @@ def post_to_instant_articles_api(content_pk):
                 source,
                 fb_page_id,
                 fb_api_url,
-                fb_token_path)
+                fb_token_path,
+                fb_dev_mode,
```

Comment:
`fb_dev_mode` indicates that if article should be sent to the Development Mode pane of the instant articles interface, and `fb_publish` indicates that if this article should be set to published when uploaded. We could check on `if not fb_dev_mode and fb_publish` but that seems sort of confusing to me.